lib/App/Mi6/Release/BumpVersion.rakumod

```diff
@@ -88,6 +84,10 @@ method next-version($version? is copy) {
 }
 
 method bump($version) {
+    my %meta = App::Mi6::JSON.decode("META6.json".IO.slurp);
```

Comment:
Assuming `App::Mi6::JSON.decode` returns a Hash, then probably binding it to `%meta` would save copying.

Reply:
Thanks for your review.
For now, I will leave it as it is.